liquibase-integration-tests/src/test/resources/changelogs/pgsql/complete/root.changelog.xml

```diff
@@ -444,5 +444,15 @@
         </createTable>
         <addPrimaryKey tableName="nocluster_table" columnNames="id" clustered="false"/>
     </changeSet>
+    <changeSet id="table-type-test" author="mallod" dbms="oracle,mysql,h2,mariadb,postgresql,snowflake">
+        <createTable tableName="tableTypeTestTable" tableType="TEMPORARY">
```

Comment:
Out of curiosity, if I am an Oracle user and I want to create a `PRIVATE TEMPORARY` table, and I put the literal string `PRIVATE TEMPORARY` in this changeset, will it construct the expected SQL? 

I was looking at the documentation here: [https://oracle-base.com/articles/18c/private-temporary-tables-18c](https://oracle-base.com/articles/18c/private-temporary-tables-18c).

Reply:
I think that wouldn't be an issue. I just tried to simplify using a common value for all specified databases. I think we could use any value from the available for the supported DBs, but it would require us to define different changesets as the used value could be supported for a given DB, but it could not for any other.